apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -136,16 +136,7 @@ const BookingPage = ({
       return router.push({
         pathname: "/success",
         query: {
-          date,
-          type: eventType.id,
-          eventSlug: eventType.slug,
-          username: profile.slug,
-          reschedule: !!rescheduleUid,
-          name: bookingForm.getValues("name"),
-          email: bookingForm.getValues("email"),
-          location: responseData.location,
-          eventName: profile.eventName || "",
-          bookingId: id,
+          uid,
```

Comment:
Same needs to be done for the recurringMutation a few lines below. We are now getting 'page not found' when booking a recurring meeting

Reply:
Hi @CarinaWolli! Thank you for the review and good catch. I have added an extra commit for the recurringMutation to also use the UID. Could you recheck that page?


@JeroenReumkens I also noticed you added two properties (periodStartDate and periodEndDate) but were not in the db select, so I added them as well